app/javascript/controllers/polygon_search_controller.js

```diff
@@ -31,7 +30,7 @@ export default class extends Controller {
     return document.getElementById("address-container")
   }
 
-  getConsultationNeighbourAddressesForm() {
+  getHiddenAddressesField() {
     return document.getElementById("addresses-hidden")
   }
```

Comment:
Probably not worth changing this here, but rather than looking up elements with `getElementById` we could also make them [Stimulus targets](https://stimulus.hotwired.dev/reference/targets), something like `data-polygon-search-target="hidden-addresses"`, which is then available as `this.hiddenAddressesTarget` and avoids having an extra function.